server/utils/utils.ts

```diff
@@ -0,0 +1,74 @@
+import R from 'ramda'
+import crypto from 'crypto'
+import moment, { Moment } from 'moment'
+
+export const isNilOrEmpty = R.either(R.isEmpty, R.isNil)
+export const getIn = R.path
+export const { equals } = R
+export const firstItem = R.head
+export const mergeWithRight = R.mergeDeepRight
```

Comment:
think we can get rid of `mergeWithRight` as not being used

Reply:
and `getIn`

Reply:
good idea. I didn't check for their use across that app